include/swift/AST/Decl.h

```diff
@@ -6818,38 +6842,11 @@ class ConstructorDecl : public AbstractFunctionDecl {
 
   ParamDecl **getImplicitSelfDeclStorage() { return &SelfDecl; }
 
-  /// Specifies the kind of initialization call performed within the body
-  /// of the constructor, e.g., self.init or super.init.
-  enum class BodyInitKind {
```

Comment:
Oh, saw the insertion of this before the deletion. Didn't realize it wasn't a pure addition.